dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithEuTaxonomyNonFinancials.cy.ts

```diff
@@ -123,7 +121,7 @@ describe("Component test for the Eu-Taxonomy-Non-Financials view page", () => {
       });
       cy.get("body").type("{esc}");
 
-      getCellValueContainer("Aligned Activities", 2).click();
+      getCellValueContainer("Aligned Activities", 2).first().click();
```

Comment:
reminder to me

Reply:
only click() leads to an error as cypress said there were more than one element to click. in main it was not the case. i have no strong reasoning about why this could change due to decollapsing.